### PR TITLE
Update batch deposit script for post-shapella deposits

### PR DESCRIPTION
**Pull request by k1rill-fedoseev**
As deposits have migrated from mGNO to GNO, few modifications are required for the deposit script to work properly again.

Docs at https://docs.gnosischain.com/node/guide/validator/deposit#option-2-direct-interaction-with-contracts should be updated after this PR as well

### CODE REVIEW

**Review by dapplion — DISMISSED** [review]
Looks good thanks @k1rill-fedoseev :heart:

File: scripts/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:14
+FROM node:20
```
Comment (akolotov) [inline scripts/Dockerfile:1]: It looks like versions of installed packages need to be updated as well:
```
41.65 error /app/node_modules/websocket/node_modules/bufferutil: Command failed.
41.65 Exit code: 1
41.65 Command: node-gyp-build
41.65 Arguments: 
41.65 Directory: /app/node_modules/websocket/node_modules/bufferutil
41.65 Output:
41.65 node:events:492
41.65       throw er; // Unhandled 'error' event
41.65       ^
41.65 
41.65 Error: spawn node-gyp ENOENT
41.65     at ChildProcess._handle.onexit (node:internal/child_process:286:19)
41.65     at onErrorNT (node:internal/child_process:484:16)
41.65     at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
41.65 Emitted 'error' event on ChildProcess instance at:
41.65     at ChildProcess._handle.onexit (node:internal/child_process:292:12)
41.65     at onErrorNT (node:internal/child_process:484:16)
41.65     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
41.65   errno: -2,
41.65   code: 'ENOENT',
41.65   syscall: 'spawn node-gyp',
41.65   path: 'node-gyp',
41.65   spawnargs: [ 'rebuild' ]
41.65 }
41.65 
41.65 Node.js v20.5.1
41.65 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
42.17 warning Error running install script for optional dependency: "/app/node_modules/utf-8-validate: Command failed.
42.17 Exit code: 1
42.17 Command: node-gyp-build
42.17 Arguments: 
42.17 Directory: /app/node_modules/utf-8-validate
42.17 Output:
42.17 node:events:492
42.17       throw er; // Unhandled 'error' event
42.17       ^
42.17 
42.17 Error: spawn node-gyp ENOENT
42.17     at ChildProcess._handle.onexit (node:internal/child_process:286:19)
42.17     at onErrorNT (node:internal/child_process:484:16)
42.17     at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
42.17 Emitted 'error' event on ChildProcess instance at:
42.17     at ChildProcess._handle.onexit (node:internal/child_process:292:12)
42.17     at onErrorNT (node:internal/child_process:484:16)
42.17     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
42.17   errno: -2,
42.17   code: 'ENOENT',
42.17   syscall: 'spawn node-gyp',
42.17   path: 'node-gyp',
42.17   spawnargs: [ 'rebuild' ]
42.17 }
42.17 
42.17 Node.js v20.5.1"
42.17 info This module is OPTIONAL, you can safely ignore this error
42.17 warning Error running install script for optional dependency: "/app/node_modules/bufferutil: Command failed.
42.17 Exit code: 1
42.17 Command: node-gyp-build
42.17 Arguments: 
42.17 Directory: /app/node_modules/bufferutil
42.17 Output:
42.17 node:events:492
42.17       throw er; // Unhandled 'error' event
42.17       ^
42.17 
42.17 Error: spawn node-gyp ENOENT
42.17     at ChildProcess._handle.onexit (node:internal/child_process:286:19)
42.17     at onErrorNT (node:internal/child_process:484:16)
42.17     at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
42.17 Emitted 'error' event on ChildProcess instance at:
42.17     at ChildProcess._handle.onexit (node:internal/child_process:292:12)
42.17     at onErrorNT (node:internal/child_process:484:16)
42.17     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
42.17   errno: -2,
42.17   code: 'ENOENT',
42.17   syscall: 'spawn node-gyp',
42.17   path: 'node-gyp',
42.17   spawnargs: [ 'rebuild' ]
42.17 }
42.17 
42.17 Node.js v20.5.1"
42.17 info This module is OPTIONAL, you can safely ignore this error
------
Dockerfile:9
--------------------
   7 |     COPY package.json yarn.lock ./
   8 |     
   9 | >>> RUN yarn install
  10 |     
  11 |     COPY truffle-config.js .
--------------------
ERROR: failed to solve: process "/bin/sh -c yarn install" did not complete successfully: exit code: 1
```

Comment (k1rill-fedoseev) [reply scripts/Dockerfile]: Updated Dockerfile to fix the issue

Comment (akolotov) [reply scripts/Dockerfile]: Thanks, it works now

**Review by dapplion — APPROVED** [review]

**Pull request merged**